src/containers/ImageIndexContainer.js

```diff
@@ -0,0 +1,23 @@
+import { connect } from 'react-redux';
+import { selectImage, resetScore } from '../actions/actions';
+import { ImageIndex } from '../views/ImageIndex';
+
+function mapStateToProps(state) {
+  return {
+    index: state.index,
+  };
+}
+
+function mapDispatchToProps(dispatch) {
+  return {
+    selectImage: imageData => {
+      dispatch(selectImage(imageData));
+      dispatch(resetScore());
+    },
```

Comment:
this is a good example of when we would want to use the function notation of `mapDispatchToProps`. though technically we could have also defined a separate `selectImageAndResetScore` thunk, and passed that single function into an object `mapDispatchToProps`.

whichever strategy will be dependent on which you prefer. or if you're jumping onto an existing project, use the existing patterns; unless there are compelling reasons you can vocalize to shift to the better pattern.